en/getting_started/rc_transmitter_receiver.md

```diff
@@ -1,48 +1,68 @@
 
 ![Mode1-Mode2](../../images/mode1_mode2.png)
 
-> **Note** The choice of transmitter mode is largely one of taste, and must be made before you buy the transmitter.
+> **Note** The choice of mode is largely one of taste, and must be made before you buy the remote control.
   
 
-## Transmitters for Ground Vehicles
+## Remote Control Units for Ground Vehicles
 
 An Unmanned Ground Vehicle (UGV)/car minimally requires a 2 channel transmitter in order to send the values for steering and speed. Commonly transmitters set these values using a wheel and trigger, two single-axis control sticks, or a single dual-axis control stick.
 
 There is nothing to stop you using more channels/control mechanisms, and these can be very useful for engaging additional actuators and autopilot modes.
 
 
-## Choosing Transmitter/Receivers
+## Choosing RC System Components
 
-You will need to select a transmitter and receiver that are compatible with each other. Compatible radio systems are often sold together. For example, this [Taranis X9D and FrSky X8R](https://hobbyking.com/en_us/frsky-2-4ghz-accst-taranis-x9d-plus-and-x8r-combo-digital-telemetry-radio-system-mode-2.html?___store=en_us) is a popular combination.
+You will need to select a remote control unit, transmitter and receiver that are compatible with each other. In addition, receivers have to be be compatible with PX4 and the flight controller hardware (below we list some of the receivers that have been validated for use with Flight Controller boards and PX4).
 
 > **Note** Before you can use the radio system you will need to *bind* them so that they communicate. Follow the binding instructions that come with your specific transmitter/receiver!
 
-In addition, receivers have to be be compatible with PX4 and the flight controller hardware. The sections below list some of the receivers that have been validated for use with Flight Controller boards and PX4
+Compatible radio systems are often sold together. For example, [FrSky Taranis X9D and FrSky X8R](https://hobbyking.com/en_us/frsky-2-4ghz-accst-taranis-x9d-plus-and-x8r-combo-digital-telemetry-radio-system-mode-2.html?___store=en_us) are a popular combination.
+
+
+### Remote Control Units
+
+One of the most popular radio control units is the *FrSky Taranis X9D*. It offers high flexibility thanks to the highly customizable open source OpenTX Firmware and a custom radio transmitter module slot. It can directly be used with the internal transmitter module when binding for example to the recommended *FrSky X4R-SB* (S-BUS, low delay) or *X4R* (PPM-Sum, legacy) receivers.
+
+![Taranis X9D Transmitter](../../images/frsky_taranis_x9d_transmitter.jpg)
+
+> **Note** The remote control unit can display vehicle telemetry when used with [FrSky](../peripherals/frsky_telemetry.md) radio modules. 
+
+Turnigy remote control units are also popular, and can be used with FrSky radio modules.
+
+### Transmitters

```

Comment:
Transmitter and receiver modules usually come in pairs because most of them are not compatible (at least across the brand).

Maybe we name this title **Transmitter/Receiver pairs**?